webview-ui/src/RetinaCapture/RetinaCapture.module.css

```diff
@@ -0,0 +1,65 @@
+.content {
+    display: grid;
+    grid-template-columns: 6rem 50rem;
+    grid-gap: 1rem;
+    align-items: center;
+}
+
+.filterContent {
```

Comment:
The vast majority of these classes are unused. As far as I can tell, we only need `.content` and `.label`.